src/components/core/Breadcrumb/Breadcrumb.tsx

```diff
@@ -26,18 +25,13 @@ const Breadcrumb = ({ breadcrumbs, pathname, rootLabel, separator }: Props) => {
 
   const breadcrumbItems = breadcrumbs ?? pathnameBreadcrumbs;
 
-  const classNames = breadcrumb();
-
   return (
-    <panda.div className={classNames.root}>
+    <panda.div>
```

Comment:
not sure what it is but the text for breadcrumbs dont render appropriately (text color)

Reply:
I fixed this, check recent